src/main/java/Deadline.java

```diff
@@ -0,0 +1,17 @@
+public class Deadline extends Task {
+
+    protected String byDate;
+
+    public Deadline(String description, String byDate) {
+        super(description);
+        this.byDate=byDate;
```

Comment:
There should be whitespace within a statement, so consider "this.byDate = byDate", similarly for line 10 and 14.